app/src/main/res/layout/content_privacy_dashboard.xml

```diff
@@ -148,10 +148,11 @@
                 app:layout_constraintTop_toTopOf="parent" />
 
             <androidx.appcompat.widget.SwitchCompat
+                tools:ignore="AndroidSwitchInXml"
                 android:id="@+id/privacyToggle"
                 android:layout_width="wrap_content"
                 android:layout_height="26dp"
-                android:theme="@style/SwitchTheme"
+                android:theme="@style/Widget.DuckDuckGo.Switch.White"
```

Comment:
Looks like this toggle lost its styling (is now blue).
![blue_toggle](https://user-images.githubusercontent.com/7963079/191965781-f82dfc63-cff8-4691-b9ed-183495d47c52.png)

Reply:
nice catch, I'll revert!